solutionbox/inception/datalab_solutions/inception/_model.py

```diff
@@ -316,12 +316,26 @@ def build_prediction_graph(self):
 
     # To extract the id, we need to add the identity function.
     keys = tf.identity(keys_placeholder)
+    labels = self.labels + ['UNKNOWN']
```

Comment:
This is hard. The model assumes there is a default label, so we have to add a default label to match the number. If 'UNKNOWN' already exists, there will be two 'UNKNOWN's, which seems fine? Any meaning label we choose may exist already in the list.